### PR TITLE
small ch 8 wording fixes

### PR DESCRIPTION
**Pull request by nfoss2**
This PR contains a few wording changes for the ch 8 sections Spatial Analysis and the Conclusion.

### CODE REVIEW

**Comment by larskotthoff** [comment]
This chapter is in the copy edit stage, so I'll leave it up to @RaphaelS1 whether to merge this.

**Comment by RaphaelS1** [comment]
Thanks will keep this open and will use it for my editing later but not merging

File: book/special.qmd [hunk 1]
```diff
@@ -787,6 +787,7 @@ Below we only show classification examples but regression follows trivially.
 ```{r special-044, message=FALSE, warning=FALSE}
 library(mlr3spatial)
 library(mlr3spatiotempcv)
+# NOTE: The `crs` argument stands for "Coordinate Reference System"
```
Comment (RaphaelS1) [inline book/special.qmd:790]:
```suggestion
```

File: book/special.qmd [hunk 2]
```diff
@@ -851,7 +852,7 @@ The resampling methods we have added can be categorized into:
 * Feature space clustering -- Clusters based on feature space and not necessarily spatiotemporal
 * Custom (partitioning) -- Grouped by factor variables
 
-The choice of method may depend on specific characteristics of the dataset and there is no easy rule to pick one method over another, full details of different methods can be found in @Schratz2021 -- the paper deliberately avoids recommending one method over another as the 'optimal' choice is highly dependent on the predictive task, autocorrelation in the data, and the spatial structure of the sampling design.
+The choice of method may depend on specific characteristics of the dataset and there is no easy rule to pick one method over another, full details of different methods can be found in @Schratz2021 -- the paper deliberately avoids recommending one method over another as the 'optimal' choice because it is highly dependent on the predictive task, autocorrelation in the data, and the spatial structure of the sampling design.
```
Comment (RaphaelS1) [inline book/special.qmd:855]:
```suggestion
The choice of method may depend on specific characteristics of the dataset and there is no easy rule to pick one method over another, full details of different methods can be found in @Schratz2021 -- the paper deliberately avoids recommending one method over another as the 'optimal' choice is highly dependent on the predictive task, autocorrelation in the data, and the spatial structure of the sampling design.
```

File: book/special.qmd [hunk 3]
```diff
@@ -913,9 +914,9 @@ plot(pred, col = c("#440154FF", "#443A83FF", "#31688EFF",
 
 ## Conclusion
 
-In this chapter, we explored going beyond deterministic regression and classification to see how functions in the `r mlr3` can be used to implement other machine learning tasks.
-Cost-sensitive classification extends the 'normal' classification setting by assuming errors associated with false negatives and false positives are unequal.
-Running cost-sensitive classification experiments is possible just using features in `r mlr3`.
+In this chapter, we explored going beyond deterministic regression and classification to see how functions in `r mlr3` can be used to implement other machine learning tasks.
+Cost-sensitive classification extends the 'normal' classification setting by assuming the cost associated with false negatives and false positives are unequal.
+Running cost-sensitive classification experiments is possible through using features in `r mlr3`.
```
Comment (RaphaelS1) [inline book/special.qmd:919]:
```suggestion
Running cost-sensitive classification experiments is possible using only features in `r mlr3`.
```

**Pull request merged**